spyder/utils/introspection/utils.py

```diff
@@ -211,6 +211,30 @@ def get_keywords(lexer):
     return keywords
 
 
+def get_words_file(file_in):
+    """
+    Completion inline
```

Comment:
Docstrings are sentences and need to end in a period. Also we can improve the one line description from `Completion inline` to something like `Extract all words from a source code file to be used in code completion.`